scripts/prepare_env_standalone.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
I don't like this `standalone` name

Reply:
I think it will be removed anyway when endpoint for checking for single tenant mode will be presented